codalab/worker/docker_image_manager.py

```diff
@@ -282,6 +283,7 @@ def download():
                             image_spec, ex
                         )
                         logger.error(failure_msg)
+                        capture_message(failure_msg)
```

Comment:
Do we need this line? We're already logging it via `logger.error` -- is that automatically sent to Sentry?

Reply:
Oh that's right. I read [this](https://docs.sentry.io/platforms/python/guides/logging/).

Reply:
Can we replace all instances of `capture_message` with `logger.error` then? I don't see the point of using `capture_message` when we can just log them so that they are sent both to the console and Sentry.